pkg/api/v1/types.go

```diff
@@ -2316,7 +2316,9 @@ type ExportOptions struct {
 // ListOptions is the query options to a standard REST list call.
 type ListOptions struct {
 	unversioned.TypeMeta `json:",inline"`
-
+	// MatchedLabels for reverse lookup.
```

Comment:
Will need better user documentation (e.g., matched against by pod selectors by default). Also, don't refer to the go field name.